authapi/serializers.py

```diff
@@ -51,11 +51,46 @@ class UserSerializer(serializers.ModelSerializer):
         many=True, source='seedteam_set', read_only=True)
     organizations = OrganizationSummarySerializer(
         many=True, source='seedorganization_set', read_only=True)
-    email = serializers.EmailField(source='username')
-    admin = serializers.BooleanField(source='is_superuser')
+    email = serializers.EmailField()
+    admin = serializers.BooleanField(source='is_superuser', required=False)
```

Comment:
Ah, I see. I think what would be ideal would be for those fields to not be required for creating users, but to require them for updating users, though that is probably more work than it it is worth. Not very important, happy for things to stay as is.